spec/function.dd

```diff
@@ -594,6 +596,7 @@ $(H2 $(LNAME2 property-functions, Property Functions))
         Foo f;
 
         f.data = 3;        // same as f.data(3);
+        f.data += 3;       // same as f.data(fdata() + 3);
```

Comment:
The semantic interpretation of `f.data+=3` ==> `f.data(f.data()+3)` is not true if the getter returns by `ref`. So this needs to be specified and clarified.